app/components/bread-crumbs.js

```diff
@@ -35,7 +40,7 @@ export default Component.extend({
   routeHierarchy: computed('currentUrl', 'currentRouteName', 'reverse', {
     get() {
       get(this, 'currentUrl');
-      const currentRouteName = getWithDefault(this, 'currentRouteName', false);
+      const currentRouteName = this.currentRouteName ? this.currentRouteName : false;
```

Comment:
I fixed some getWithDefault deprecations but the real fix would be to replace it with an addon that's actually maintained :smile:.